scholia/api.py

```diff
@@ -284,8 +284,8 @@ def entity_to_doi(entity):
     for statement in entity['claims'].get('P356', []):
         doi = statement['mainsnak']['datavalue']['value']
         return doi
-    else:
-        return ''
+
+    return ''
```

Comment:
The documentation to the function says that the function returns an empty string if the field is not set.